ci/setup.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+git fetch origin master:master
+git diff --name-only master $HEAD > modified
+echo "these files have changed"
+cat modified
```

Comment:
if this is a huge file , it will basically do a long print with cat, I wouldn't suggest to print it out in a setup as it's not a part of setup.